wagtail/images/templates/wagtailimages/images/index.html

```diff
@@ -13,10 +13,13 @@
             targetOutput: "#image-results"
         }
 
+        const dropdownSelectors = ['#collection_chooser_collection_id', "#order_images_by"];
         $(function() {
-            $('#collection_chooser_collection_id').on('change', function() {
-                this.form.submit();
-            })
+            dropdownSelectors.forEach(dropdownSelector =>
```

Comment:
one to check with the UI team. 
I would've thought we're trying to move away from jQuery, so changes like this are the perfect place to try to drop jQuery-related usage.

I believe something like:

```js
const dropdownSelectors = ['#collection_chooser_collection_id', "#order_images_by"];
dropdownSelectors.forEach(dropdownSelector =>
    document.querySelector('body').addEventListener('click', function(event) {
        if (event.target.matches(dropdownSelector)) {
            this.form.submit();
        }
    }
);
```
should do it (⚠️ untested)

Reply:
I have added [a commit to replace jQuery usage](https://github.com/wagtail/wagtail/pull/8430/commits/cc88da7832a964772610cb43691f5ae86a2a03ae).

Reply:
I have added [a commit to replace jQuery usage](https://github.com/wagtail/wagtail/pull/8430/commits/cc88da7832a964772610cb43691f5ae86a2a03ae).